warehouse/config.py

```diff
@@ -135,7 +135,21 @@ class RootFactory:
                 Permissions.APIObservationsAdd,
             ),
         ),
-        (Allow, Authenticated, "manage:user"),
+        (
+            Allow,
+            Authenticated,
+            (
+                Permissions.Account2FA,
+                Permissions.AccountAPITokens,
+                Permissions.AccountManage,
+                Permissions.AccountManagePublishing,
+                Permissions.AccountVerifyEmail,
```

Comment:
just a thought, but some of these feel a bit too granular but overall seem reasonable to me. i think the general question i'd ask when granularizing is "would another entity do this on the behalf of another". in this instance i think that most of the Verify* will only be acted on by a user so in my (very negotiable) mental model we could just have "Permissions.AccountVerify" that covers all of these.

probably no action here just writing down the thought provoked.

Reply:
I hear you - I was going through and breaking things down to the smallest unit first, and then seeing where things can maybe be combined. I'll keep these in mind when I take a more holistic view of account provisioning, and may bundle many of them together then.